src/ansys/acp/core/_tree_objects/model.py

```diff
@@ -412,6 +424,79 @@ def export_materials(self, path: _PATH) -> None:
                 )
             )
 
+    @supported_since("25.1")
```

Comment:
would it make sense to have an argument modeling_plies and export all plies if it is None, or "All". This would allow the user to export only a specific set. The export can become expensive if the surface of many plies is exported.

Reply:
Agreed; should we implement this in this PR, or just put it in the backlog?

I was mainly pulling these changes from the `adaptive_mesh_workflow` feature branch for the sake of the `supported_since` changes.

Reply:
I would change it here because it would otherwise cause a minimal change in the API.